### PR TITLE
The queue was not being declared properly, and became a global property.

### PR DESCRIPTION
**Pull request by jkodumal**
Same issue for the requestor and update_processor. I'll push this out to the integration harnesses for testing as well.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -45,14 +46,13 @@ var new_client = function(sdk_key, config) {
...
 
-  queue = [];
-
   if (!sdk_key && !config.offline) {
     throw new Error("You must configure the client with an SDK key");
   }
 
   if (!config.use_ldd && !config.offline) {
-    requestor = Requestor(sdk_key, config);
+    var requestor = Requestor(sdk_key, config);
```
Comment (apucacao) [inline index.js]: minor: we should move these bindings up to the top of the scope to avoid a similar issue in the future.

**Comment by apucacao** [comment]
👍 , with 1 comment

**Comment by apucacao** [comment]
👍

**Comment by jkodumal** [comment]
This passes integration harness-- merging and releasing version 3.0.4.

**Pull request merged**